tests/js/list-tests.js

```diff
@@ -1348,7 +1348,7 @@ module.exports = {
     testListNestedFromJSON: function() {
         let json = '{"id":1, "name": [{ "family": "Larsen", "given": ["Hans", "Jørgen"], "prefix": [] }, { "family": "Hansen", "given": ["Ib"], "prefix": [] }] }';
         let parent = JSON.parse(json);
-        const realm = new Realm({schema: [schemas.ParentObjectLocal, schemas.NameObjectLocal]});
+        const realm = new Realm({schema: [schemas.ParentObject, schemas.NameObject]});
```

Comment:
I introduced `ParentObjectLocal` and `NameObjectLocal` to make the test easier. `ParentObject` and `NameObject` are for sync and have primary keys.

Reply:
let me change it quick